packages/domainproperties/src/models.ts

```diff
@@ -799,6 +808,12 @@ export class DomainField extends Record({
         return isFieldNew(this);
     }
 
+    isValid(): boolean {
+        // TODO should the rest of these checks move up to the getErrors() function and return different FieldErrors?
+        // if so, then we can remove this isValid() function and just use !hasErrors()
```

Comment:
Not sure if you saw this TODO message that I added. Thoughts?

Reply:
Yes, I didn't get time to address this, but I left the message there to address in the next branch.